packages/miniflare/src/index.ts

```diff
@@ -1281,6 +1288,18 @@ export class Miniflare {
 				if (!colors$.enabled) message = stripAnsi(message);
 				this.#log.logWithLevel(logLevel, message);
 				response = new Response(null, { status: 204 });
+			} else if (url.pathname === "/browser/launch") {
+				// Version should be kept in sync with the supported version at https://github.com/cloudflare/puppeteer?tab=readme-ov-file#workers-version-of-puppeteer-core
+				const puppeteer = await npxImport(
+					"puppeteer@22.8.2",
+					this.#log.warn.bind(this.#log)
+				);
+
+				// @ts-expect-error Puppeteer is dynamically installed, and so doesn't have types available
```

Comment:
I believe you can add `puppeteer` as a dev-dependency and then grab the type of the import, without impacting the distributable size:

```
type Puppeteer = typeof import('puppeteer')
```

Reply:
the types clash a bit if I do this? @penalosa is this why we didn't do it? :)

Reply:
same with this one. @penalosa @petebacondarwin we can move the discussion in a small wiki if you don't mind merging it this way?